services/spar/src/Spar/Types.hs

```diff
@@ -29,6 +29,10 @@ import qualified Data.ByteString.Builder as Builder
 import qualified Data.Text as ST
 
 
+data Void
+
+type BindCookie = SimpleSetCookie "wire.com"
```

Comment:
Can we make name the cookie sth else plz, sth along the lines of `bind` or `zbind` or some other variant? (to keep in line with `zuid`.
Also, let's also restrict the cookie domain, which probably needs changes in the [web-saml-sso](https://github.com/wireapp/saml2-web-sso) ?

Reply:
`zbind` it is.

fixing the domain is simple enough, but i already have `setCookieSameSite == True`, isn't that enough?  if you don't respond i'll just do both.

Reply:
Done.  See also: https://github.com/wireapp/saml2-web-sso/blob/5758cb355a1a9956bb5d3ab2402f82b003a5a961/src/SAML2/WebSSO/Cookie.hs#L57-L76